### PR TITLE
Remove showcase page

### LINKED ISSUE #3805: Showcase page is very outdated - Remove page
Whilst prepping the masterclass, I noticed that the [showcase](https://vanillaframework.io/showcase) page is very outdated - a lot of the screenshots feature old designs or suru strips, conjure-up now redirects to juju.is, and one of them (Notes on development) doesn't use Vanilla anymore. 

If we're making Vanilla more of an internal tool, maybe the page is redundant?

FireShot Capture 032 - Showcase - Vanilla documentation - 0 0 0 0

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- Removed showcase page
- Removed links to it, and added a redirect (back to the homepage)

Fixes #3805 

## QA

- Try to visit https://vanilla-framework-3810.demos.haus/showcase
- See that you're taken back to the homepage

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

- [x] PR should have one of the following labels to automatically categorise it in release notes:
  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
- [x] Vanilla version in `package.json` should be updated relative to the [most recent release](https://github.com/canonical-web-and-design/vanilla-framework/releases/latest), following semver convention:
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
- [x] Any changes to component class names (new patterns, variants, removed or added features) should be listed on the [component status page](https://github.com/canonical-web-and-design/vanilla-framework/blob/master/templates/docs/component-status.md).
- [x] Documentation side navigation should be updated with the relevant labels.

### CODE REVIEW

File: redirects.yaml [hunk 1]
```diff
@@ -6,3 +6,4 @@
 /css/(?P<path>.*): /static/css/{path}
 /js/(?P<path>.*): /static/js/{path}
 /docs/patterns/article-pagination: /docs/patterns/pagination
+/showcase: /
```
Comment (bartaz) [inline redirects.yaml:9]: Great little touch!

**Review by bartaz — APPROVED** [review]

File: redirects.yaml (hunk 1 above)
Comment (anthonydillon) [reply redirects.yaml:9]: Shouldn't this path return deleted? As it is removed content not moved.

Comment (sowasred2012) [reply redirects.yaml:9]: @anthonydillon oh - hadn't considered that, I thought it was better to send users _somewhere_, plus I didn't know about `deleted.yaml` till just now. I'll get another PR up for that.

Comment (anthonydillon) [reply redirects.yaml:9]: Oh it's definitely better than letting it 404 but being honest is always best :)

**Pull request merged**